app/Actions/GenerateAppKey.php

```diff
@@ -17,12 +17,17 @@ public function __construct(Shell $shell)
 
     public function __invoke()
     {
-        $this->logStep('Running php artisan key:generate');
+        app('console-writer')->logStep('Setting APP_KEY in .env');
 
-        $process = $this->shell->execInProject('php artisan key:generate');
+        $process = $this->shell->execInProject(sprintf("php artisan key:generate%s", $this->withQuiet()));
 
         $this->abortIf(! $process->isSuccessful(), 'Failed to generate application key successfully', $process);
```

Comment:
Failed to generated `APP_KEY` successfully

Reply:
Done